Applications/SystemMonitor/ProcessModel.h

```diff
@@ -89,14 +89,12 @@ class ProcessModel final : public GUI::Model {
     virtual GUI::Variant data(const GUI::ModelIndex&, Role = Role::Display) const override;
     virtual void update() override;
 
-    struct CpuInfo
-    {
+    struct CpuInfo {
         u32 id;
-        float total_cpu_percent{0.0};
-        HashMap<String, String> values;
```

Comment:
this wasn't being used anywhere, so I removed it for now - but since all the values are in a JSON object anyway, accessing them is now a lot easier once we need to do that. :^)